### ISSUE TITLE
Mail unexpectedly quits

### ISSUE THREAD

**Issue by farhatizakaria** [elementary/mail#817]
### What Happened?

When I browse my emails, it just crashs and stops then quits

### Steps to Reproduce

1. Opend the mail app
2. Scrolled and checked my emails
3. It crashed then quits for no reason

### Expected Behavior

It supposed to run smothly like Geary app

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

### Log Output

```shell
(process:2576): GLib-GIO-DEBUG: 13:33:39.000: _g_io_module_get_default: Found default implementation dconf (DConfSettingsBackend) for ?gsettings-backend?
(process:2576): dconf-DEBUG: 13:33:39.000: watch_fast: "/io/elementary/mail/" (establishing: 0, active: 0)
(process:2576): GLib-DEBUG: 13:33:39.000: unsetenv() is not thread-safe and should not be used after threads are created
(io.elementary.mail:2576): dconf-DEBUG: 13:33:39.002: watch_established: "/io/elementary/mail/" (establishing: 1)
(io.elementary.mail:2576): dconf-DEBUG: 13:33:39.486: sync
```

**Comment by TomiOhl** [comment]
Probably a duplicate of #814 
Or do you experience it even when opening an unread mail? It only occurs for me 2-3 seconds after opening an unread message (i.e when it attempts to mark it as read)

**Comment by farhatizakaria** [comment]
@TomiOhl yeah, but then it just quits without 'not responding' message

**Comment by lenemter** [comment]
Should be fixed by #825
